opta/core/terraform.py

```diff
@@ -952,11 +952,26 @@ def force_unlock(cls, layer: "Layer", *tf_flags: str) -> None:
             print("Terraform Lock Id could not be found.")
             return
 
-        nice_run(
-            ["terraform", "force-unlock", *tf_flags, lock_id],
-            use_asyncio_nice_run=True,
-            check=True,
-        )
+        try:
+            nice_run(
+                ["terraform", "force-unlock", *tf_flags, lock_id],
+                use_asyncio_nice_run=True,
+                check=True,
+            )
+        except Exception as e:
+            logger.info(
+                "An exception occured while removing the Terraform Lock. Trying to Remove the lock forcefully."
```

Comment:
the part about `Trying to Remove the lock forcefully.` should only be logged if we are about to do it (aws/gcp), can it be moved inside that function?